spec/services/articles/creator_spec.rb

```diff
@@ -51,6 +51,7 @@
     let(:invalid_attributes) { attributes_for(:article) }
 
     before do
+      invalid_attributes[:title] = Faker::Book.title
```

Comment:
It's a little confusing that we have a valid title in the `invalid_attributes` but the processing of the markdown is also a bit confusing at least to me so I can deal with this. Maybe someone else will have a better suggestion who understands this flow better.

Reply:
`invalid_attributes` has many valid attributes, the only invalid one is `body_markdown`.

Reply:
We could change this from `invalid_attributes` to `invalid_body_attributes` to make this more explicit.